### DISCUSSION TITLE
AxisBottom dates differ from tooltip

### DISCUSSION THREAD

**Discussion by libnine** [airbnb/visx#1067] [Q&A] [answered]
Hey all - having an issue that I'm a bit stumped on. My dates are inconsistent with my tooltip. Seems to be shifted by 1. Any ideas on why this may be?

Sandbox: [https://codesandbox.io/s/quirky-shockley-qbjrb?file=/src/App.tsx](url)

**Comment by williaster** [comment, marked as answer]
Hey @libnine this is because of the `transform` which you are applying to tick labels. In effect you've shifted them over by one bar. See here how the tick `<g />` has the tick on the correct bar but the label is shifted one.

<img src="https://user-images.githubusercontent.com/4496521/107809535-1e51f200-6d20-11eb-96c4-a22fcca9c17b.png" width="300" />

`tickLabelProps` accepts any `TextProps` from [`@visx/text`](https://airbnb.io/visx/docs/text), including `angle` which will rotate the text appropriately without offsetting the origin. [Updated sandbox](https://codesandbox.io/s/young-platform-omtvy?file=/src/App.tsx:5289-5312).

<img src="https://user-images.githubusercontent.com/4496521/107809248-a08de680-6d1f-11eb-89db-2257defb9c98.png" width="300" />
